core/src/main/scala/slamdata/SbtSlamDataBase.scala

```diff
@@ -366,11 +374,23 @@ abstract class SbtSlamDataBase extends AutoPlugin {
         } else {
           file.delete()
         }
+      },
+
+      // TODO make this suck less
+      trickleGithubIsAutobumpPullRequest := { pr =>
+        pr.title == "Applied dependency updates" &&
```

Comment:
I suggest extracting the title as a val, to avoid the two usages getting out of sync.